### ISSUE TITLE
Fig hijacks the user's shell if it is PowerShell; switches to zsh, with no obvious way to revert the change

### ISSUE THREAD

**Issue by mklement0** [withfig/fig#598]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue

### Issue Details

Related: #237

Installing Fig while [PowerShell](https://github.com/PowerShell/PowerShell/blob/master/README.md) (`/usr/local/bin/pwsh`) is the default shell (as reflected in env. var. `SHELL`) causes all terminal applications to switch to `zsh` on startup, with no obvious way to revert the change (the value of `SHELL` doesn't change).

### Environment

```bash
macOS 10.16
```

```none
Error: EOF
Fig Version:  [?]
UserShell: /usr/local/bin/pwsh
Bundle path:
Autocomplete: unknown
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility:
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path:
iTerm Integration:
Hyper Integration:
VSCode Integration:
Docker Integration:
Symlinked dotfiles:
Only insert on tab:
Installation Script:
PseudoTerminal Path:
SecureKeyboardInput:
SecureKeyboardProcess:
Current active process:
Current working directory: /Users/mklement/Desktop/pg
Current window identifier:
Path: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/mklement/.fig/bin
```

**Comment by mschrage** [comment]
Hey @mklement0! This sounds very annoying. Thanks so much for reporting this. We don't currently support PowerShell, so I'm not exactly sure what mechanism is triggering this, but will investigate.

**Comment by mklement0** [comment]
Thanks, @mschrage. Let me know if there's something I can do to help.

**Comment by SeparateRecords** [comment]
As far as I can see this only happens if your login shell is powershell

**Comment by SeparateRecords** [comment]
Got a fix for this, will be resolved along with #881

**Comment by SeparateRecords** [comment]
Fix is merged!
